### PR TITLE
Rename Manifest to Config

### PR DESCRIPTION
**Pull request by samuelgoto**
***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/369.html" title="Last updated on Dec 3, 2022, 12:59 AM UTC (7ebe1b0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/369/aa3a1a6...7ebe1b0.html" title="Last updated on Dec 3, 2022, 12:59 AM UTC (7ebe1b0)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -653,11 +652,11 @@ Sec-Fetch-Dest: webidentity
 ```
 </div>
 
-The file is parsed expecting [=Manifest=] JSON object.
+The file is parsed expecting [=Config=] JSON object.
```
Comment (npm1) [inline spec/index.bs:655]:
```suggestion
The file is parsed expecting a [=Config=] JSON object.
```

File: spec/index.bs [hunk 2]
```diff
@@ -1251,19 +1250,19 @@ To <dfn>fetch the accounts list</dfn> given a [=Manifest=] |manifest| and an {{I
...
-When <dfn>computing the manifest URL</dfn> given an {{IdentityProviderConfig}} |provider| and a
-[=string=] |manifestString|, perform the following steps:
+When <dfn>computing the config URL</dfn> given an {{IdentityProviderConfig}} |provider| and a
+[=string=] |configString|, perform the following steps:
     1. Let |configUrl| be the result of running [=url parser=] with |provider|'s
         {{IdentityProviderConfig/configURL}}.
-    1. Let |manifestUrl| be the result of running [=url parser=] given |manifestString|.
-    1. If |manifestUrl| is failure, let |manifestUrl| be the result of running [=url parser=] given
-        |manifestString| (the relative URL) and |configUrl| (the base URL).
+    1. Let |configUrl| be the result of running [=url parser=] given |configString|.
```
Comment (npm1) [inline spec/index.bs:1257]: You are naming this exactly the same as above, and they are not. Please revert.

File: spec/index.bs [hunk 3]
```diff
@@ -1617,19 +1616,19 @@ separate section for [[#privacy]] considerations.
 ## Content Security Policy ## {#content-security-policy}
 <!-- ============================================================ -->
 
-The first fetches triggered by the FedCM API are the manifest list, which is public, and the
-internal manifest. Imagine a malicious script included by (and running as) the [=RP=] attempting to
+The first fetches triggered by the FedCM API are the well-known file, which is public, and the
+internal config. Imagine a malicious script included by (and running as) the [=RP=] attempting to
```
Comment (npm1) [inline spec/index.bs:1620]: Remove internal?

File: spec/index.bs [hunk 4]
```diff
@@ -1251,19 +1250,19 @@ To <dfn>fetch the accounts list</dfn> given a [=Manifest=] |manifest| and an {{I
 </div>
 
 <div algorithm>
-When <dfn>computing the manifest URL</dfn> given an {{IdentityProviderConfig}} |provider| and a
-[=string=] |manifestString|, perform the following steps:
+When <dfn>computing the config URL</dfn> given an {{IdentityProviderConfig}} |provider| and a
```
Comment (npm1) [inline spec/index.bs:1253]: This name is now confusing as it implies we are computing configURL, whereas it is not.

**Review by npm1 — CHANGES_REQUESTED** [review]

File: spec/index.bs [hunk 5]
```diff
@@ -2102,14 +2103,14 @@ To <dfn>maybe fetch the accounts list</dfn> given a [=Manifest=] |manifest| and
 </div>
 
 <div algorithm>
-To <dfn>Sign-in to the IDP</dfn> given a [=Manifest=] |manifest| and an {{IdentityProviderConfig}}
+To <dfn>Sign-in to the IDP</dfn> given a [=Config=] |config| and an {{IdentityProviderConfig}}
```
Comment (yi-gu) [inline spec/index.bs:2106]: same here

**Comment by samuelgoto** [comment]
This is no longer needed, because @npm1 renamed things in his latest PR. Closing this.

**Pull request closed without merging**